app/poc-common/clinical-services/config.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('poc.common.clinicalservices')
+    .config(config);
+
+  config.$inject = ['$provide'];
+
+  function config($provide) {
```

Comment:
Is this needed?

Reply:
Will be when we start using authorization.